WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1738,12 +1737,21 @@ private extension EditableOrderViewModel {
         orderSynchronizer.removeCoupon.send(code)
     }
 
-    func addFee(_ formattedFeeLine: String) {
-        let feeLine = OrderFactory.newOrderFee(total: formattedFeeLine)
-        orderSynchronizer.setFee.send(feeLine)
+    func addFee(with total: String, name: String? = nil) {
+        let feeLine = OrderFactory.newOrderFee(total: total, name: name)
```

Comment:
The `OrderFactory.newOrderFee()` method initializes all new fee lines as `.taxable ` by default, is this something we want to be able to change from here based on store settings?

Reply:
Yeah, for now, we make them all taxable but we will make it customizable in a future PR.